packages/marketplace/contracts/TransferManager.sol

```diff
@@ -210,12 +210,7 @@ abstract contract TransferManager is Initializable, ITransferManager {
         bool isBundle = nftSide.asset.assetType.assetClass == LibAsset.AssetClass.BUNDLE;
 
         if (isBundle) {
-            if (!_isTSBSeller(nftSide.account)) {
-                remainder = _doTransfersWithFeesAndRoyaltiesForBundle(paymentSide, nftSide, nftSideRecipient);
-            } else {
-                // No royalties but primary fee should be paid on the total value of the bundle
-                remainder = _transferProtocolFees(remainder, paymentSide, fees);
-            }
```

Comment:
We don't want to eliminate the `_isTSBSeller ` check because TSB may be selling land, catalysts etc and we dont want to pay royalty ourselves.

Reply:
Generally:

`_isTSBSeller` => forces primary market on all bundle items, skips royalties
`_isTSBBundleSeller` => allows selling non-primary market items in a bundle, pays royalties
`regular user` => can only sell items that pass `_isPrimaryMarket` check as true, no royalties

Reply:
The names of the roles probably aren't great, I find them bit confusing.

Reply:
Roles name will be updated to 

1. TSB_PRIMARY_MARKET_SELLER
2. TSB_SECONDARY_MARKET_SELLER